docs/data_structure_design.md

```
@@ -11,38 +11,38 @@
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)